src/ProjectService.Business/Commands/DeleteRoleCommand.cs

```diff
@@ -0,0 +1,35 @@
+﻿using LT.DigitalOffice.Kernel.AccessValidatorEngine.Interfaces;
+using LT.DigitalOffice.Kernel.Exceptions;
+using LT.DigitalOffice.ProjectService.Business.Commands.Interfaces;
+using LT.DigitalOffice.ProjectService.Data.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Business.Commands
+{
+    public class DeleteRoleCommand : IDeleteRoleCommand
```

Comment:
DisableRole, because we nothing not delete in database

Reply:
Renamed everywhere